recipes/pymonetdb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymonetdb" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "77e3bacd4a947bd54419939cdc4b4dcdee579edaad3a898c53c11b30695f3be9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - pymonetdb
+    - pymonetdb.control
+
+about:
+  home: http://www.monetdb.org
+  license: MPL-2.0
+  license_family: NONE
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
```

Comment:
Please raise an issue upstream to include the license file in the `MANIFEST.in` for future packaging.

Reply:
[done](https://github.com/gijzelaerr/pymonetdb/issues/25)